packages/@sanity/form-builder/src/inputs/Array/ItemValue.js

```diff
@@ -191,18 +189,21 @@ export default class RenderItemValue extends React.Component<Props> {
           key={item._key}
           title="Edit"
           onAction={this.handleDialogAction}
-          actions={[{
-            index: '1',
-            name: 'close',
-            title: 'Close'
-          },
-          {
-            index: '2',
-            name: 'delete',
-            kind: 'simple',
-            title: 'Delete',
-            color: 'danger'
-          }
+          showCloseButton={false}
+          actions={[
```

Comment:
Could you hoist the actions into a constant? No need for creating these on each render, and a bit easier on the eye, IMO.

Reply:
👍